scripts/release/audit_rosdistro.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python3
+
+# Copyright 2014-2020 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import argparse
+import copy
+import sys
+
+from ros_buildfarm.argument import add_argument_arch
+from ros_buildfarm.argument import add_argument_build_name
+from ros_buildfarm.argument import add_argument_cache_dir
+from ros_buildfarm.argument import add_argument_config_url
+from ros_buildfarm.argument import add_argument_os_code_name
+from ros_buildfarm.argument import add_argument_os_name
+from ros_buildfarm.argument import add_argument_rosdistro_name
+from ros_buildfarm.common import filter_blocked_dependent_package_names
+from ros_buildfarm.common import get_cached_package_manifests
+from ros_buildfarm.common import get_os_package_name
+from ros_buildfarm.common import Target
+from ros_buildfarm.config import get_index as get_config_index
+from ros_buildfarm.config import get_release_build_files
+from ros_buildfarm.package_repo import get_package_repo_data
+from ros_buildfarm.status_page import get_jenkins_job_urls
+
+from rosdistro import get_distribution_file
+from rosdistro import get_cached_distribution
+from rosdistro import get_index
+
+
+def main(argv=sys.argv[1:]):
+    parser = argparse.ArgumentParser(
+        description='Check if the sync criteria are matched to sync ' +
```

Comment:
The description seems to be copy-n-pasted from somewhere else.

Nitpick: no need for `+`.